### ISSUE TITLE
Android support

### ISSUE THREAD

**Issue by ivanmoskalev** [ivanmoskalev/react-native-compressed-jsbundle#3]

**Comment by tux2nicolae** [comment]
I would be interested in this as well :)

**Comment by ivanmoskalev** [comment]
@tux2nicolae Do you have any specific usecase in mind (for example, Android Instant), or are you interested in general case size reduction?

**Comment by tux2nicolae** [comment]
@ivanmoskalev right now we are using your library on IOS for app clips but we would also be interested in being able to use it with Android instant as well. Indeed on android the size limits are a little bit more permissive around 15MB in download size but a react-native on Android also should ship the JS engine with the application, which increases the app size.

Either way, I would definitely trade the decompression time for even one MB of download size on Android too.

**Comment by tux2nicolae** [comment]
You're right, and there is one more aspect to this, on android the size is counted as the download size **which is already compressed**. I didn't actually measured the real size win if we'll use Brotli for compressing the jsbundle.
